src/core/core.helpers.js

```diff
@@ -53,6 +53,16 @@ module.exports = function(Chart) {
 		}
 		return base;
 	};
+    // Adjust element index when min is defined
+	helpers.adjustIndex = function(element, chartConfig) {
+		var min = chartConfig.options.scales.xAxes[0].ticks.min;
+		var labels = chartConfig.data.labels;
+
+		if (!element._adjustedIndex && min) {
+			element._index -= labels.indexOf(min);
+			element._adjustedIndex = true;
+		}
+	};
```

Comment:
Could this method directly be on the [element class](/chartjs/Chart.js/blob/master/src/core/core.element.js#L91) instead?

```javascript
    adjustIndex: function(config) {
        var min = config.options.scales.xAxes[0].ticks.min;
        if (this._adjustedIndex || min === undefined) {
            return;
        }

        this._index -= config.data.labels.indexOf(min);
        this._adjustedIndex = true;
    }
```